mkdocs_multirepo_plugin/util.py

```diff
@@ -7,14 +7,12 @@
 from sys import platform, version_info
 from typing import Any, Dict
 
-from mkdocs.utils import warning_filter
-
 # used for getting Git version
 GitVersion = namedtuple("GitVersion", "major minor")
 LINUX_LIKE_PLATFORMS = ["linux", "linux2", "darwin"]
 
+# This is a global variable imported by other modules
 log = logging.getLogger("mkdocs.plugins." + __name__)
-log.addFilter(warning_filter)
```

Comment:
Can you briefly explain to me why we're getting rid of this filter?

Reply:
The filter is according to the mkdocs logging a NOP and will be removed isoon:

```
INFO    -  DeprecationWarning: warning_filter doesn't do anything since MkDocs 1.2 and will be removed soon. All messages on the `mkdocs` logger get counted automatically.
```